vite.config.ts

```diff
@@ -19,6 +20,20 @@ export default ({ mode }: { mode: 'production' | 'development' | 'test' }) => {
     }),
   ];
 
+  // These files are only needed in dev / test, don't include in prod builds
+  if (['1', 'true'].includes(process.env.INCLUDE_EPG_DATA?.toLowerCase() || '')) {
```

Comment:
@dbudzins Danny, do we need this additional flag? Maybe we can just use mode === 'production' not to include it. Or do we still need for preview builds?

Reply:
We need it for preview/demo builds. I've actually refactored it a bit more to make better use of 'mode' that way, but didn't want to complicate this PR any further.